lib/rpmsg/rpmsg.c

```diff
@@ -29,7 +29,7 @@ static uint32_t rpmsg_get_address(unsigned long *bitmap, int size)
 
 	nextbit = metal_bitmap_next_clear_bit(bitmap, 0, size);
```

Comment:
Maybe add a comment above that the bitmap is used to allocate addresses above RPMSG_RESERVED_ADDRESSES. It took me some time to see how it worked.